data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/TargetType.java

```diff
@@ -7,12 +7,7 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import org.opensearch.dataprepper.model.event.DataType;
-import org.opensearch.dataprepper.typeconverter.TypeConverter;
-import org.opensearch.dataprepper.typeconverter.IntegerConverter;
-import org.opensearch.dataprepper.typeconverter.StringConverter;
-import org.opensearch.dataprepper.typeconverter.DoubleConverter;
-import org.opensearch.dataprepper.typeconverter.BooleanConverter;
-import org.opensearch.dataprepper.typeconverter.LongConverter;
+import org.opensearch.dataprepper.typeconverter.*;
```

Comment:
Will need to change this back to not using * imports. You can make this happen by default if you are using Intellij

Reply:
Reverted this change now 👍 Also fixed in my Intelij to not do this going forward